json-schema-core/build.gradle

```diff
@@ -0,0 +1,101 @@
+ plugins {
+     id 'java'
+     id 'checkstyle'
+     id 'com.github.spotbugs'
+ }
+
+ group = 'io.ballerina'
+ version = project.version
+
+ repositories {
+     mavenCentral()
```

Comment:
is this correct,
Don't we needs to add mavenLocal() and other github packages.
Otherwise it is harder to debug

Reply:
I think it's highly unlikely that developers of this tool will do a lang fix and use `publishToMavenLocal`. We can always add if required, but no harm adding it now. 

(Note that this is not required to look up from the cache)

Reply:
https://medium.com/decisionbrain/when-to-not-use-mavenlocal-in-your-gradle-build-script-6da03902f9df